### PR TITLE
[oneDNN] Enable bf16 kernels for FusedBatchNormV3

### PR DESCRIPTION
**Pull request by gaurides**
If Eigen FusedBatchNormV3 kernel is used in a model on CPU, and oneDNN optimizations are disabled; then it will crash. We had observed such crash in 3 models. To fix the crash, this PR enables valid Eigen kernels for FusedBatchNormV3 and Grad for CPU and also enables the relevant unit-tests. 
Also addresses [this old issue](https://github.com/tensorflow/tensorflow/issues/57888 )

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @penpornk Can you please review this PR ? Thank you!

**Comment by gaurides** [comment]
@penpornk - I don't know what is the failure for Py+CPP job. Looks like there was some timeout, not related to my changes.

File: tensorflow/core/kernels/fused_batch_norm_op.cc [hunk 1]
```diff
@@ -1963,6 +1963,18 @@ REGISTER_KERNEL_BUILDER(Name("FusedBatchNormGradV3")
...
+                            .Device(DEVICE_CPU)
+                            .TypeConstraint<bfloat16>("T")
+                            .TypeConstraint<float>("U"),
+                        FusedBatchNormOpV3<CPUDevice, bfloat16, float>);
+
+REGISTER_KERNEL_BUILDER(Name("FusedBatchNormGradV3")
+                            .Device(DEVICE_CPU)
+                            .TypeConstraint<bfloat16>("T")
+                            .TypeConstraint<float>("U"),
+                        FusedBatchNormGradOpV3<CPUDevice, bfloat16, float>);
```
Comment (penpornk) [inline tensorflow/core/kernels/fused_batch_norm_op.cc:1977]: Nit: Please add one blank line after this line.

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR!

File: tensorflow/core/kernels/fused_batch_norm_op.cc (hunk 1 above)
Comment (gaurides) [reply tensorflow/core/kernels/fused_batch_norm_op.cc:1976]: done.

**Review by penpornk — APPROVED** [review]
Thank you!

**Comment by penpornk** [comment]
These 3 CI failures are unrelated
* [Py+CPP Ubuntu CPU](https://source.cloud.google.com/results/invocations/d9fa27f1-ca9a-4b76-ad3b-08851ad96c1f/log): Tests timed out.
* [Py+CPP Ubuntu GPU](https://source.cloud.google.com/results/invocations/5af8fd45-1f55-4a1e-b963-c507082eb812/log): Test timed out.
* [AMD ROCm](http://ml-ci.amd.com:21096/blue/organizations/jenkins/tensorflow%2Fgithub-prs-upstream-master%2FAMD-ROCm-Community-CI-Build/detail/PR-60413/6/pipeline): This PR doesn't modify XLA.
```
ERROR: /workspace/tensorflow/compiler/xla/service/gpu/BUILD:853:11: in deps attribute of cc_library rule //tensorflow/compiler/xla/service/gpu:gpu_executable: Label '//tensorflow/tsl/platform:random' is duplicated

ERROR: /workspace/tensorflow/compiler/xla/service/gpu/BUILD:853:11: Analysis of target '//tensorflow/compiler/xla/service/gpu:gpu_executable' failed
```

**Pull request merged**